README.md

```diff
@@ -167,6 +167,16 @@ if #available(iOS 13.0, *) {
 ```
 ***Note*** Flag is available on iOS 13 and above versions
 
+### Modify network requests
+
+You can track and modify network requests throughout OktaOidc by passing object that conforms to `OktaNetworkRequestCustomizationDelegate` protocol to the `requestCustomizationDelegate` property of `OktaOidcConfig` instance.
+
+```swift
+let configuration = OktaOidcConfig(with: {YourOidcConfiguration})
+configuration.requestCustomizationDelegate = {YourDelegateInstance}
```

Comment:
Please add example how to use delegate methods

Reply:
Done